examples/kafka/architect.yml

```diff
@@ -3,39 +3,67 @@ name: examples/kafka
 parameters:
   TOPIC_NAME:
     default: architect
+  KAFKA_LISTENERS:
+    default: INTERNAL://:9092,EXTERNAL://:9093
+  KAFKA_ADVERTISED_LISTENERS:
+    default: INTERNAL://:9092,EXTERNAL://${{ services.kafka.interfaces.external.host }}:${{ services.kafka.interfaces.external.port }}
+  KAFKA_LISTENER_SECURITY_PROTOCOL_MAP:
+    default: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+  KAFKA_ADVERTISED_HOST_NAME:
```

Comment:
What is KAFKA_ADVERTISED_HOST_NAME?

Reply:
That's overridden for local and is the host address. I'll change it in the config to be the address people using macOS would use though